pkg/scaffold/role.go

```diff
@@ -167,6 +167,12 @@ rules:
   - secrets
   verbs:
   - "*"
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
+  verbs:
+  - get
```

Comment:
This is required by Helm's tiller implementation, which performs a `GET` on the CR namespace during the helm release installation to see if the namespace exists.

We could put this behind a conditional option in the template, but this seems pretty innocuous to me.

Thoughts?

Reply:
This is fine. So long as a Role grants the permissions to GET its own namespace.
https://github.com/operator-framework/helm-app-operator-kit/pull/64#discussion_r230803841